### PR TITLE
Code Builder Beta Doc update

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
Removed waitlist information and consolidated "Important Considerations" to a single topic.

### What does this PR do?
Made an update to Code Builder docs regarding waitlists. 
#<Insert GitHub Issue>, @W-12545655@

### Functionality Before
<insert gif and/or summary>

### Functionality After
<insert gif and/or summary>

### CODE REVIEW

File: docs/_articles/en/codebuilder/about.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ You can install Code Builder as a managed package in any supported Salesforce or
 ### Important Considerations for Code Builder Beta
 * We've capped usage for beta at 20 hours for a maximum of 30 days. 
 * We highly recommend that you save your work and close the browser tab that is running Code Builder to stop the usage clock when you aren’t using Code Builder.
-* Code Builder is in open beta and is available to try on a first-come, first-served basis. If no spots are available, you're added to a waitlist, and receive an email when a spot opens up for you.
+* Throughout the beta, Code Builder environments could be deleted. All beta environments will be removed before GA.
```
Comment (ekapner) [inline docs/_articles/en/codebuilder/about.md:21]: Can we elaborate more about why the environments would be deleted? Or Throughout the beta, we might delete Code Builder environments with low usage. or due to capacity balancing.

File: docs/_articles/en/codebuilder/cb-start.md [hunk 2]
```diff
@@ -4,9 +4,7 @@ lang: en
 ---
 
 ## Overview
-Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment.
-## Important Considerations
-We've capped usage for beta at 20 hours for a maximum of 30 days. We highly recommend that you save your work and close the browser tab that is running Code Builder to stop the usage clock when you aren’t using it.
+Complete this Quick Start to become familiar with the Code Builder interface and perform some simple tasks in your Code Builder environment. Before you start, see [Code Builder Overview](./en/codebuilder/about) for more information and important considerations.
```
Comment (ekapner) [inline docs/_articles/en/codebuilder/cb-start.md:7]: Do you want them to do the latter before they go to the Quick Start? Or are you just providing a list of "quick start" content? 
Before you begin:

- Complete to the Quick Start to yadda yadda
- Before you dive deeper, see Code Builder Overview for blah blah

**Review by ekapner — COMMENTED** [review]
A couple suggestions. If you need me to approve, ping me in Slack.

File: docs/_articles/en/codebuilder/cb-start.md (hunk 2 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/codebuilder/cb-start.md:7]: We just want them to read the overview and look at the important considerations section. We were sort of duplicating the text from the "Important Considerations" section here. I just consolidated it.

File: docs/_articles/en/codebuilder/about.md (hunk 1 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/codebuilder/about.md:21]: Not low usage, capping usage due to cost of hosting. I don't think we want to say that.

**Review by ekapner — APPROVED** [review]

**Pull request merged**